dask/dataframe/core.py

```diff
@@ -331,6 +331,15 @@ def npartitions(self):
         """Return number of partitions"""
         return len(self.divisions) - 1
 
+    @property
+    def attrs(self):
+        """Return a dictionary storing arbitrary metadata."""
```

Comment:
Can you use `derived_from(pd.DataFrame)` decorator so that we get the pandas docstring?